website/content/maintainers/templates/governance-elections.md

```diff
@@ -8,6 +8,260 @@ aliases:
- which contains instructions in markdown comments.
+| Audience of this HowTo | Audience of The Document    | Required by CNCF      |
+| ---------------------- | --------------------------- | --------------------- |
+| Maintainers            | Contributors                | Yes, graduated        |
+
+
+This HowTo is for project maintainers who need Governance documentation for their project.
+The goal of a GOVERNANCE.md file is to inform contributors about how your
+project is run, and encourage them to get involved in project leadership.
+
+Great governance docs will:
+
+* Show potential contributors that their contributions will be treated fairly
+* Show contributors that leadership positions are attainable
+* Provide a framework for decision making and resolving disagreements
+* Define a process for promoting contributors and improving maintainer continuity
+
+## Fill out the template
+
+The [GOVERNANCE-elections.md template] is located in the CNCF [project-template repository].
+
+Copy the template file into your repository, and rename it `GOVERNANCE.md`.
+
+There are instructions for filling out the template that look like the example below:
+
+![screenshot of the CONTRIBUTING.md template, there is a link to instructions, and a warning emoji with text explaining how to fill out this section of the template](../sample-instructions.png)
+
+Some links are specific to your project.
+Search for the word TODO in the markdown for links that need to be customized.
+When you finish editing the template, remove the Instruction links that explain how to fill out the template. Also remove any ⚠️ prompts and their explanatory text.
+
+# Steering Committee Election Governance
+
+This model of governance is only appropriate for very large open source projects
+with a complex structure often made up of multiple Special Interest Groups (SIGs)
+and / or Working Groups (WGs) where a steering committee or similar elected body
+is required to make project-wide decisions that span across multiple SIGs / WGs
+or to make decisions when multiple groups are in disagreement. It may also be
+used as a way to enforce employer representation limits to ensure that a project
+has leadership representation from a variety or organizations.
+
+This model was based mostly on a simplified verion of the [Kubernetes governance]
+and another example of this model is the [Knative governance].
+
+# Is This Template For Us?
+
+An election-based steering committee governance model is for you if your project:
+
+1) Is organized into distinct groups, like SIGs / WGs, that might have competing 
+priorities
+2) Wants to enforce employer representation limits for leadership
+3) Has a large number of maintainers (20+) who cannot easily govern themselves
+
+If you don’t yet meet these conditions, we recommend that you start with a
+simpler governance template, like the [Maintainer Council] template, and move
```

Comment:
suggestion: Link to the guide for maintainer council at https://contribute.cncf.io/maintainers/templates/governance-maintainer/ since it has explanatory text on how to know if it's the right template for you. In general, I lean towards always linking people to the guide for a template rather than directly to the template itself since I think it will give them the additional context they need (and the rest of the website links/nav) to help them figure out what to do with it.

Reply:
Excellent idea - I've fixed this link in the most recent commit.